Tensile/KernelWriter.py

```diff
@@ -3333,6 +3333,13 @@ def kernelBody( self, kernel, tensorParametersA, tensorParametersB ):
         kl.append(self.comment("LocalSplitU: local write"))
         kl.append(self.localSplitULocalWrite(kernel))
 
+        # Use non-MI pass for MI + LSU>1
+        backup = kernel["EnableMatrixInstruction"]
+        if backup:
+          serializedStoreBackup = self.serializedStore
+          kernel["EnableMatrixInstruction"] = False
```

Comment:
> Temporarily changing the kernel parameter off and on seems like it could be error prone, or confusing when debugging. If some of the localSplitU functions are dependent on EnableMatrixInstruction could we instead make a separate internal parameter that defiunes this parameter combo that these functions could check?

Global store code is huge and it is very difficult to go through non-MatrixInstruction code with keeping EnableMatrixInstruction = true.
I thought adding a lot of changes to force MI+LSU to go through non-MFMA path could cause unexpected fails...

Reply:
> > Temporarily changing the kernel parameter off and on seems like it could be error prone, or confusing when debugging. If some of the localSplitU functions are dependent on EnableMatrixInstruction could we instead make a separate internal parameter that defiunes this parameter combo that these functions could check?
> 
> Global store code is huge and it is very difficult to go through non-MatrixInstruction code with keeping EnableMatrixInstruction = true. I thought adding a lot of changes to force MI+LSU to go through non-MFMA path could cause unexpected fails...

I will check the code to see if I can create a new parameter like EnableMatrixInstructionStore or something like that.
Plus, probably I can set self.serializedStore=False at the beginning for MFMA+LSU and no backup needed for self.serializedStore.

Reply:
Updated